lms/djangoapps/lti_provider/tests/test_management.py

```diff
@@ -0,0 +1,87 @@
+"""
+Test lti_provider management commands.
+"""
+
+
+from django.test import TestCase
+from mock import patch
+from opaque_keys.edx.keys import UsageKey, CourseKey
+
+from student.tests.factories import UserFactory
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from xmodule.modulestore.tests.utils import TEST_DATA_DIR
+from xmodule.modulestore.xml_importer import import_course_from_xml
+
+from lti_provider.management.commands import rescore_lti
+from lti_provider.models import GradedAssignment, LtiConsumer, OutcomeService
+
+class CommandArgsTestCase(TestCase):
+    """
+    Tests management commands parse arguments properly
+    """
+    def test_command_arguments(self):
+        cmd = rescore_lti.Command()
+        parser = cmd.create_parser('./manage.py', 'rescore_lti')
+        args = parser.parse_args(['course-v1:edX+test_course+2525_fall', 'UBC/Law281/2015_T1'])
+        self.assertEqual(len(args.course_keys), 2)
+        key = args.course_keys[0]
+        self.assertIsInstance(key, CourseKey)
+        self.assertEqual(unicode(key), 'course-v1:edX+test_course+2525_fall')
+
+
+class RescoreLTITestCase(SharedModuleStoreTestCase):
+    """
+    Test manage.py rescore_lti command
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(RescoreLTITestCase, cls).setUpClass()
+        cls.course_key = cls.store.make_course_key('edX', 'lti_provider', '3000')
+        course_items = import_course_from_xml(
+            cls.store,
+            'test_user',
+            TEST_DATA_DIR,
+            source_dirs=['simple'],
+            static_content_store=None,
+            target_id=cls.course_key,
+            raise_on_failure=True,
+            create_if_not_present=True,
+        )
+        cls.course_items = course_items
+        cls.course = course_items[0]
```

Comment:
These 2 class variables (`course_items` and `course`) don't seem to be used elsewhere so can remove these 2 lines.